circle.yml

```diff
@@ -0,0 +1,6 @@
+test:
+  pre:
+  - ls -al ~/.pyenv/shims ~/.pyenv/versions 
+  - pyenv global 2.6.8 2.7.9 pypy-2.4.0 3.2.5 3.3.3 3.4.1
```

Comment:
Ok, no problem. I'm just trying to do a minimal review to lighten the burden of repo maintainer.